Makefile.toml

```diff
@@ -206,13 +206,11 @@ exec --fail-on-error gradle test  --warning-mode all
 
 [tasks.test-nanobind-feature]
 category = "Tests"
-dependencies = ["build-feature"]
+dependencies = ["build-feature", "gen-nanobind-feature"]
```

Comment:
issue: gen and test should be invoked separately; it should be possible to test the files in their current state (e.g. if you're making local edits).